sql-cli/pyproject.toml

```diff
@@ -39,7 +39,7 @@ Jinja2 = ">=2.10.1, <=3.1"
 python-frontmatter = "^1.0.0"
 PyYAML = ">=5.4.1"
 python-dotenv = "^0.21.0"
-astro-sdk-python = {extras = ["all"], version = "^1.2.0"}
+astro-sdk-python = {extras = ["all"], version = "^1.3.0.dev1"}
```

Comment:
Does this work? 1.3.0.dev1 isn't a release right?

Reply:
@pankajkoti I've modified how this all works so your local dev uses the local python-sdk. We won't be able to release 0.3 until python SDK 1.3.0 comes out.

Reply:
I am also wondering how this would work 🤔 

Did you test this in a new python venv? I understand that this works within the project, but don't quite understand how this would work outside of the project.